### PR TITLE
Implementation homework of Lesson 2 is finished

### PR DESCRIPTION
**Pull request by shmatko1095**
Bash-script for add .c files to archive implemented 
@a-vodka please review

### CODE REVIEW

**Comment by a-vodka** [comment]
- Please, define temp dir as variable, and check if it exist
- Clear temp dir after archiving

File: 01-Git_Bash_Make/script.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+tmpDir=$tmpDir"tmp/guesanumber"
+realiseDir=$realiseDir"realise"
```
Comment (a-vodka) [inline 01-Git_Bash_Make/script.sh]: tmpDir and realiseDir have to be upper-case

File: 01-Git_Bash_Make/script.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+if [ -d "${tmpDir}" ]; then
+   mkdir -p $realiseDir
+   cp *.c $tmpDir
+   tar -zcf $realiseDir/guesanumber.tar.gz  -C  $tmpDir *.c
+   
+   rm ${tmpDir}/*.c
+   rmdir -p $tmpDir
+else
+   echo "dir can not be created"
+fi
```
Comment (a-vodka) [inline 01-Git_Bash_Make/script.sh]: Please, add empty line at end of the file

**Comment by a-vodka** [comment]
You have fixed only second node. What about first one?

**Comment by shmatko1095** [comment]
Fixed, @a-vodka , please review

**Pull request closed without merging**